source/entities/MapLight.cpp

```diff
@@ -20,15 +20,12 @@
 #include "entities/Tile.h"
 #include "game/Seat.h"
 #include "gamemap/GameMap.h"
-#include "modes/ModeManager.h"
 #include "network/ODPacket.h"
 #include "network/ODServer.h"
 #include "network/ServerNotification.h"
 #include "render/RenderManager.h"
-#include "render/ODFrameListener.h"
-#include "utils/Helper.h"
 #include "utils/LogManager.h"
-#include "utils/Random.h"
+//#include "utils/Random.h"
```

Comment:
Yeah, there was some code I commented out about the light flickering, and that's part of it. I wasn't sure if I should fully remove it or not, so I left it commented out for now.